packages/gatsby-plugin-gatsby-cloud/.gitignore

```diff
@@ -2,6 +2,6 @@ yarn.lock
 /*.js
 !index.js
 
-components/
+/components/
```

Comment:
We were previously ignoring files in `components` and `src/components`. This tells git to track files within the nested directory named `components`